SurfaceDevCenterManager/DevCenterApi/DevCenterHandler.cs

```diff
@@ -231,6 +231,56 @@ public async Task<DevCenterResponse<Submission>> GetSubmission(string ProductId,
             return retval;
         }
 
+        private const string DevCenterPartnerSubmissionUrl = "/hardware/products/relationships/sourcepubliherid/{0}/sourceproductid/{1}/sourcesubmissionid/{2}";
```

Comment:
Typo: sourcepubli(s)herid?

Reply:
Typo on the service side - I will let the Dev Center team know